GTG/gtk/editor/editor.py

```diff
@@ -65,12 +67,12 @@ def __init__(self,
         self.deletebutton = self.builder.get_object("delete_editor")
         self.deletebutton.set_tooltip_text(GnomeConfig.DELETE_TOOLTIP)
         self.subtask_button = self.builder.get_object("insert_subtask")
-        self.subtask_button.set_tooltip_text(GnomeConfig.SUBTASK_TOOLTIP)
-        self.inserttag_button = self.builder.get_object("inserttag")
-        self.inserttag_button.set_tooltip_text(GnomeConfig.TAG_TOOLTIP)
-        self.open_parents_button = self.builder.get_object("open_parents")
-        self.open_parents_button.set_tooltip_text(
-            GnomeConfig.OPEN_PARENT_TOOLTIP)
+        # self.subtask_button.set_tooltip_text(GnomeConfig.SUBTASK_TOOLTIP)
```

Comment:
subtask button doesn't change it's tooltip. Delete menu item doesn't change it's tooltip. They should be defined in .ui file

I'm also of the opinion, that there should be multiple buttons (mark as done, mark as undone). You should not modify those buttons, just choose which one you decide to choose. Put the description how the element should look like into .ui file and the logic of when to show what in python. You can do it very easily:

```python
if condition:
  button_a.show()
  button_b.hide()
else:
  button_a.hide()
  button_b.show()
```

The same thing for menu items